package.json

```diff
@@ -3999,6 +3999,426 @@
+														"type": "command",
+														"command": "references-view.findReferences"
+													},
+													{
+														"key": "S",
+														"name": "Go to symbol in project",
+														"icon": "project",
+														"type": "command",
+														"command": "workbench.action.showAllSymbols"
+													}
+												]
+											},
+											{
+												"key": "s",
+												"name": "+Send/REPL",
+												"icon": "repl",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "b",
+														"name": "Execute block or selection in REPL",
+														"icon": "selection",
+														"type": "command",
+														"command": "language-julia.executeCodeBlockOrSelection"
+													},
+													{
+														"key": "c",
+														"name": "Execute code cell in REPL and move",
+														"icon": "export",
+														"type": "command",
+														"command": "language-julia.executeCellAndMove"
+													},
+													{
+														"key": "d",
+														"name": "Change directory here",
+														"icon": "folder-opened",
+														"type": "command",
+														"command": "language-julia.cdHere"
+													},
+													{
+														"key": "f",
+														"name": "Execute file in REPL",
+														"icon": "file",
+														"type": "command",
+														"command": "language-julia.executeFile"
+													},
+													{
+														"key": "i",
+														"name": "Start REPL",
+														"icon": "repl",
+														"type": "command",
+														"command": "language-julia.startREPL"
+													},
+													{
+														"key": "m",
+														"name": "Execute code in REPL and move",
+														"icon": "debug-continue",
+														"type": "command",
+														"command": "language-julia.executeCodeBlockOrSelectionAndMove"
+													},
+													{
+														"key": "s",
+														"name": "Execute code in REPL",
+														"icon": "play",
+														"type": "command",
+														"command": "language-julia.executeJuliaCodeInREPL"
+													},
+													{
+														"key": "C",
+														"name": "Connect external REPL",
+														"icon": "vm-connect",
+														"type": "command",
+														"command": "language-julia.connectREPL"
+													},
+													{
+														"key": "D",
+														"name": "Stop REPL",
+														"icon": "stop",
+														"type": "command",
+														"command": "language-julia.stopREPL"
+													},
+													{
+														"key": "F",
+														"name": "Execute active file in REPL",
+														"icon": "file",
+														"type": "command",
+														"command": "language-julia.executeActiveFile"
+													}
+												]
+											},
+											{
+												"key": "x",
```

Comment:
On a second thought, there are a few candidates for this key.

In spacemacs, 
- `x` used more commonly for kill
- `c` used more commonly for clear/change
- `d` used more commonly for delete

Seems like `c` might be a better choice? Do you have a preference? If we were to change it to `c`, we probably should also change the `d` and `D` in the clear menu to `c` and `C`.